test/MerkleProof.js

```diff
@@ -0,0 +1,60 @@
+var MerkleProof = artifacts.require("./MerkleProof.sol");
+
+import { sha3 } from "ethereumjs-util";
+import MerkleTree from "./helpers/merkleTree.js";
+
+contract('MerkleProof', function(accounts) {
+  let merkleProof;
+
+  before(async function() {
+    merkleProof = await MerkleProof.new();
+  });
+
+  describe("verifyProof", function() {
+    it("should return true for a valid Merkle proof", async function() {
+      const elements = ["a", "b", "c", "d"].map(el => sha3(el));
```

Comment:
See the comment above. Shouldn't the `MerkleTree` class be responsible for initially hashing the `elements`?